components/catalogs/Makefile

```
@@ -66,8 +66,8 @@ catalogs-unit-back:
 	APP_ENV=test $(PHP_RUN) vendor/bin/phpunit \
 		--configuration $(CATALOGS_PATH)/back/ \
 		--bootstrap $(CATALOGS_BOOTSTRAP_PATH) \
-		--testsuite Catalogs_Unit \
-		--log-junit var/tests/phpunit/phpunit_catalogs_unit.xml
+		--log-junit var/tests/phpunit/phpunit_catalogs_unit.xml \
+		--testsuite Catalogs_Unit ${O}
```

```suggestion
		--testsuite Catalogs_Unit ${O}
```

I'm honestly against it.
Having arguments like `${0}` in the makefile make it really hard to understand and maintain.
Either we stop using the makefile and use a better alternative, or we could also create shortcuts.
I wouldn't mind something like `components/catalogs/bin/phpunit` with predefined args.